web/src/components/Toast.tsx

```diff
@@ -7,15 +7,19 @@ type Props = {
   description?: string;
   isCloseable?: boolean;
   toastAction?: () => void;
+  toastClose?: () => void;
   toastActionText?: string;
 };
 
-function Toast(props: Props) {
+function Toast({ title, description, toastAction, toastActionText, toastClose }: Props) {
   const { t } = useTranslation();
-  const { title, description, toastAction, toastActionText } = props;
   const [open, setOpen] = useState(true);
   const handleToastAction = () => {
-    toastAction && toastAction();
+    toastAction?.();
```

Comment:
Is this really valid? Looks so weird 😅

Reply:
haha yeah it's valid syntax 😅

We can reconsider it but then we would always need an action which we might not want if we are planning to use this toast for other things. Like notifying the users they are offline or something similar.